### PR TITLE
7487 update team page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-7487

### CODE REVIEW

**Comment by MaximKraev** [comment]
:+1:

File: source/about/index.html.erb [hunk 1]
```diff
@@ -137,37 +170,81 @@ reading & travelling by bicycle.</p>
...
+      <p>She also dreams about a trip in the Himalayas.</p>
     </div>
   </div>
+
   <div class="member">
-    <%= image_tag "about/jessica.png" %>
-    <h2>Jessica May</h2>
-    <h4>Office Manager</h4>
+    <%= image_tag "about/dmitri.png" %>
+    <h2>Dmitri Suchkov</h2>
```
Comment (destitutus) [inline source/about/index.html.erb]: sorry, but spelling of name is wrong. correct one is `Dmitrii`

File: source/about/index.html.erb [hunk 2]
```diff
@@ -137,37 +170,81 @@ reading & travelling by bicycle.</p>
...
     <div class="description">
-      <p>Self proclaimed admin goddess by day, classical pianist by night.</p>
-      <p>Jessica likes order and harmony and thus enjoys filing, paying bills, tying up loose
-        ends and feeding the team with her latest culinary experiments to the background
-        strains of Bach, Beethoven or barking dogs.</p>
+      <p>It's hard to find a Codio subsystem that has not been fixed or improved by Dmitri.</p>
+      <p>He makes sure all our services are up & running smoothly. He enjoys writing powerful
+        yet simple code that delivers for our users.
+      </p>
+      <p>Dmitri is also interested in information security, loves meeting interesting people,
```
Comment (destitutus) [inline source/about/index.html.erb]: Comment above

**Comment by denpira** [comment]
Done

**Comment by destitutus** [comment]
👍

**Pull request merged**